### PR TITLE
chore(codeowner): added @wesleytodd as default code owner

### PR DESCRIPTION
**Pull request by wesleytodd**
Since we now have a few folks working on the project I wanted to add a bit more formality. I had already set the "require one code owner approval" setting but I forgot that I never had a code owner set. As we all work together on this I was thinking I would slowly add folks for areas they work in. 

Longer term my ideal is to get this publishing with Release Please so I am not in the way of progress. But I just wanted to have a few more checks in place before then to ensure everything runs smoothly.

### CODE REVIEW

**Review by Megapixel99 — APPROVED** [review]

**Comment by Megapixel99** [comment]
Unsure if I needed to approve, but  it said the PR was waiting for an approval...

**Comment by wesleytodd** [comment]
Yeah I just added y'all more so that you knew. Thanks though. Honestly the github ui is freaking out for me right now, so maybe they are having some issues. I just merged locally and pushed on main, but that said it was rejected despite this ui saying it was just merged.

**Pull request merged**
